pkg/virt-handler/vm.go

```diff
@@ -1462,6 +1477,19 @@ func (d *VirtualMachineController) processVmUpdate(origVMI *v1.VirtualMachineIns
 			}
 		}
 
+		// configure network
+		res, err := d.podIsolationDetector.Detect(vmi)
+		if err != nil {
+			return fmt.Errorf("failed to detect isolation for launcher pod: %v", err)
+		}
+
+		if err := res.DoNetNS(func() error {
+			return network.SetupPodNetworkPhase1(vmi, res.Pid())
+		}); err != nil {
+			return fmt.Errorf("failed to configure vmi network: %v", err)
+		}
```

Comment:
not very important, but perhaps we could separate the network configuration (this one and on lines 142-1437) to a helper method for a better clarity.

Reply:
These lines carry different error log messages which I believe is important to have separated. I prefer not to.